Makefile

```diff
@@ -17,3 +17,8 @@ build: FORCE
 
 run:
 	yarn start
+
+debug-dump:
+	git rev-parse HEAD > REVISION
+	zip -r debug-dump.zip REVISION tmp cvrs.db
```

Comment:
Maybe timestamp the zip filename? Not critical.

Reply:
Done.